Formula/a/avro-cpp.rb

```diff
@@ -1,33 +1,31 @@
 class AvroCpp < Formula
   desc "Data serialization system"
   homepage "https://avro.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=avro/avro-1.11.3/cpp/avro-cpp-1.11.3.tar.gz"
-  sha256 "fba242aef77ec819d07561fcba93751721956de8d0cae8e1f2f300b54b331bae"
+  url "https://www.apache.org/dyn/closer.lua?path=avro/avro-1.12.0/cpp/avro-cpp-1.12.0.tar.gz"
+  mirror "https://archive.apache.org/dist/avro/avro-1.12.0/cpp/avro-cpp-1.12.0.tar.gz"
+  sha256 "f2edf77126a75b0ec1ad166772be058351cea3d74448be7e2cef20050c0f98ab"
   license "Apache-2.0"
-  revision 6
 
   bottle do
-    sha256 cellar: :any,                 arm64_sequoia: "82b1a0dea57a3a31212c4276544e531efba46f7d19b45a9fc0b99a0b4a68c9d0"
-    sha256 cellar: :any,                 arm64_sonoma:  "ea66a997dbb67c668b2efeefc6c55d99bb18eba228107a80ca62eab4c0e6d616"
-    sha256 cellar: :any,                 arm64_ventura: "cb8df69dec1f3fd1fc81b92439e5f790f5d0b9b15c813c09fd67d2bce97faea8"
-    sha256 cellar: :any,                 sonoma:        "1e6a8d749991317610e3b6ea1138a8e4d76d900a3aa705b5367cae864b2ee5b4"
-    sha256 cellar: :any,                 ventura:       "bff702fbaa5e5d3804c890b944ae5f89466adab8cc589b6d69242f128bd9680f"
-    sha256 cellar: :any_skip_relocation, arm64_linux:   "9031c2d1bff4c0f2ceb59ae01efd28e0a983fd7d49eb8ab7a0e0dfbb4f8e1293"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "9b869f1c4d9cb66d3cc15dbdb24ca17ad431d457807bfc7dab3aca7fe3015831"
+    sha256 cellar: :any,                 arm64_sequoia: "291b84c78bc647e73768a8e56238854a0921e6bb4cb0719941d2224dc615d4a1"
+    sha256 cellar: :any,                 arm64_sonoma:  "b4814a1221cb3c2b269295b082a7800f9e98b6e35353b5169e0cd11bde2b1204"
+    sha256 cellar: :any,                 arm64_ventura: "5baa7aba33ff81ea4533abd7cea3618e388b297f9ed515e9070081aa18c5f682"
+    sha256 cellar: :any,                 sonoma:        "b6871bfbea04fb244cb3703fc02b5c60f9694a7d69e8651690ac871a3a2be192"
+    sha256 cellar: :any,                 ventura:       "9c38cb460359de66ad6c65bf50356a22698d9a08cf50f4ea4798a7e9acc653cb"
+    sha256 cellar: :any_skip_relocation, arm64_linux:   "44bfb408cd8eca3fdb23efbcb046e741768f4e1de5e86c62fc00f98b68e94966"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "3fa20bafdf7c160c06da0dc7fd48574cf2fb584e01f36a513f11a8fc0aef1056"
   end
 
   depends_on "cmake" => :build
   depends_on "pkgconf" => :build
   depends_on "boost"
+  depends_on "fmt" # needed for headers
```

Comment:
If this is header-only:

```suggestion
  depends_on "fmt" => [:build, :test] # needed for headers
```
as library consumers will not need it at runtime.